static/js/public/snap-details/blog-posts.js

```diff
@@ -77,8 +77,8 @@ function snapDetailsPosts(holderSelector, templateSelector, showOnSuccessSelecto
 
   blogPosts.path = snap;
 
-  blogPosts.fetch(function() {
```

Comment:
Maybe .fetch could just return a promise and not get callback as an argument, but again it's a code style comment not really related to this PR.